RELEASE_NOTES.md

```diff
@@ -1,6 +1,14 @@
 Release Notes
 =============
 
+v1.0.0-alpha.x
+--------------
+
+### Improvements
+
+- Updated to use non-deprecated version of `TraitsData`.
```

Comment:
[Fix] Can we note the increase in runtime openassetio version now required by the generated code

Reply:
How's this? https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/51/commits/722c4eca271ca23b6238b3a846f335d333a50b1e

Reply:
Cool thanks - we need to remember to revisit all the dependent projects to unify naming of the beta once we have figured out what well be able to call it to satisfy pep440